modules/decision_reviews/app/controllers/decision_reviews/v2/higher_level_reviews_controller.rb

```diff
@@ -19,8 +21,9 @@ def show
       end
 
       def create
+        req_body_obj = normalize_area_code_for_lighthouse_schema(request_body_hash)
         hlr_response_body = decision_review_service
-                            .create_higher_level_review(request_body: request_body_hash, user: @current_user,
+                            .create_higher_level_review(request_body: req_body_obj, user: @current_user,
                                                         version: 'V2')
```

Comment:
The [spec file](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/decision_reviews/spec/requests/v2/higher_level_reviews_spec.rb) for this controller is only at 41.38% test coverage. Please update the spec file.

Reply:
Thank you!